spec/features/selling/view_upcoming_deliveries_spec.rb

```diff
@@ -55,8 +55,7 @@
       end
 
       it "shows a delivery until 11:59 the day of the delivery" do
-        # Subtract 4 hours to allow for EDT
-        Timecop.travel("May 8, 2014 7:30 PM") do
+        Timecop.travel(ActiveSupport::TimeZone.new(market.timezone).local_to_utc(Time.new(2014, 5, 8, 23, 59, 00))) do
```

Comment:
@thermistor is there a cleaner way to `Timecop.travel` to a time in a specific timezone? I wasn't sure if doing `Time.zone = xxx` would pollute other subsequent tests.

Reply:
This should not pollute in my understanding. I don't know a cleaner way, I've only used Timecop on this project.